test/broadcasting.jl

```diff
@@ -4,10 +4,10 @@ using Test, DataFrames, PooledArrays, Random, CategoricalArrays
 
 const ≅ = isequal
 
-refdf = DataFrame(reshape(1.5:15.5, (3,5)))
+refdf = DataFrame(reshape(1.5:15.5, (3,5)), [:x1, :x2, :x3, :x4, :x5])
```

Comment:
What about using `:gennames` here and elsewhere?

Reply:
ok - I will change this